tox.ini

```diff
@@ -29,7 +29,7 @@ whitelist_externals =
 commands =
     - mypy xclim
     pylint --rcfile=setup.cfg --exit-zero xclim
-    pytest --nbval docs/notebooks --durations=3
+    pytest -x --nbval docs/notebooks --durations=3
```

Comment:
Do we still want to abort on first failure?

Reply:
Same as above. I don't think we want to abort on first failure.